packages/open-next/src/adapters/config/util.ts

```diff
@@ -86,10 +86,22 @@ export function loadAppPathsManifestKeys(nextDir: string) {
   const appPathsManifestJson = fs.existsSync(appPathsManifestPath)
     ? fs.readFileSync(appPathsManifestPath, "utf-8")
     : "{}";
-  const appPathsManifest = JSON.parse(appPathsManifestJson) as Record<
-    string,
-    string
-  >;
+  return JSON.parse(appPathsManifestJson) as Record<string, string>;
+}
+
+export function loadAppPathRoutesManifest(nextDir: string) {
+  const appPathRoutesManifestPath = path.join(
+    nextDir,
+    "app-path-routes-manifest.json",
+  );
+  const appPathRoutesManifestJson = fs.existsSync(appPathRoutesManifestPath)
+    ? fs.readFileSync(appPathRoutesManifestPath, "utf-8")
+    : "{}";
+  return JSON.parse(appPathRoutesManifestJson) as Record<string, string>;
```

Comment:
nit:

```suggestion
  if (fs.existsSync(appPathRoutesManifestPath)) {
    return JSON.parse(fs.readFileSync(appPathRoutesManifestPath, "utf-8"));
  }
  return {};  
```

maybe add an explicit return type to avoid having to cast